pkg/usrlocalsharelima/usrlocalsharelima.go

```diff
@@ -47,6 +48,11 @@ func Dir() (string, error) {
 		// - agent: /usr/local/share/lima/lima-guestagent.Linux-x86_64
 		// - dir:   /usr/local/share/lima
 		filepath.Join(selfDirDir, "share/lima/lima-guestagent."+ostype+"-"+arch),
+		// candidate 2: lauched by `~/go/bin/dlv dap`
+		// - self: ${workspaceFolder}/cmd/limactl/__debug_bin_XXXXXX
+		// - agent: ${workspaceFolder}/_output/share/lima/lima-guestagent.Linux-x86_64
+		// - dir:  ${workspaceFolder}/_output/share/lima
+		filepath.Join(selfDirDirDir, "_output/share/lima/lima-guestagent."+ostype+"-"+arch),
```

Comment:
This should be set only when `limactl` is inside `_output/bin`?

I also wonder if this should be set only if `--debug` is set?

Reply:
changed to add more `lima-guestagent` location candidates only when detected being built with `go-delve/delve`.

Reply:
Maybe overengineering?

Reply:
changed to checking `--debug`.